contrib/grinder/scripts/tests.py

```diff
@@ -87,12 +90,14 @@ def test_init_process(self):
     self.assertEqual(type(t1), query.QueryThread)
 
```

Comment:
I'd like a test here confirming that the first thread after the ingest and query threads is an annotation thread.